src/utils/util/resolver-pipeline/oscal-profile-resolve-modify.xsl

```diff
@@ -48,8 +48,11 @@
         </xsl:if>
     </xsl:template>
 
-    <!-- priority to override template match="control//*" -->
-    <xsl:template match="param" priority="2">
+    <!-- The match="control//*" template is responsible for inserting other
+        param elements before or after a param context node, while this template
+        is responsible for processing this param element. -->
+    <xsl:template name="process-param">
+        <xsl:context-item as="element(param)" use="required"/>
```

Comment:
Part of the bug fix to enable param to have insertions before/after AND have param set